lighthouse-cli/test/smokehouse/test-definitions/offline-local/offline-expectations.js

```diff
@@ -107,9 +107,16 @@ module.exports = [
         },
       },
       InstallabilityErrors: {
-        errors: [
-          {errorId: 'no-icon-available'},
-        ],
+        errors: {
```

Comment:
I assume the audit(s) will have to account for this starting in #11745?

Reply:
yes we'll need an update to 11745.